src/prompt.ts

```diff
@@ -141,3 +149,8 @@ function getPrompt(name: string, type?: string, defaultValue?: string) {
 
   return prompt
 }
+
+function replacePrompt(prompt: string) {
```

Comment:
Please rename this function to be more specific.  
For example `hidePrompt`, `maskPrompt`

Reply:
Well, it actually just replaces the prompt with the one you give it, assuming it is the same length. Giving it a name like `maskPrompt` would give the impression that the actual masking is happening inside that function, don't you think.

If that is what you really want, I will move the `getPrompt` call to this function and rename it.

Reply:
@mhverbakel I see what's going on.   You are in fact replacing the entire prompt with the newly specified prompt arg.  I actually think this is fine then.